plugin/src/main/java/org/gradle/testretry/internal/framework/TestNgTestFrameworkStrategy.java

```diff
@@ -59,7 +64,19 @@ public TestFramework createRetrying(JvmTestExecutionSpec spec, Test testTask, Se
                 }
             });
 
-        return new TestNGTestFramework(testTask, retriedTestFilter, instantiator, classLoaderCache);
+        VersionNumber gradleVersion = VersionNumber.parse(testTask.getProject().getGradle().getGradleVersion());
+        if (gradleVersion.getMajor() >= 6 && gradleVersion.getMinor()>= 6) {
```

Comment:
Can be simplified using `org.gradle.util.GradleVersion`:
```suggestion
        if (GradleVersion.current().compareTo(GradleVersion.version("6.6")) >= 0) {
```

Reply:
Had the same in mind - but it does not work for the `rc` version

Reply:
This should work:
```suggestion
        if (GradleVersion.current().getBaseVersion().compareTo(GradleVersion.version("6.6")) >= 0) {
```